services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet:
+    movie = Movie.objects.all()
+    if genres_ids:
+        movie = movie.filter(genres__in=genres_ids)
+    if actors_ids:
+        movie = movie.filter(actors__in=actors_ids)
+    return movie
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider using `Movie.objects.filter(id=movie_id).first()` or handling the `Movie.DoesNotExist` exception to avoid potential errors if the movie with the given ID does not exist.